docs/proxy.md

```diff
@@ -0,0 +1,43 @@
+# Running ASB behind a proxy
+
+It is recommended to review the [Openshift Documentation ](https://docs.openshift.com/container-platform/3.7/install_config/http_proxies.html)
+and to configure a cluster accordingly before attempting to run the broker behind
+a proxy. When running an Ansible Service Broker inside of a proxied Openshift cluster,
```

Comment:
Probably best to start using "Ansible Broker".

TYPO: Openshift should be OpenShift.